rasa/shared/core/slot_mappings.py

```diff
@@ -202,20 +202,23 @@ def check_mapping_validity(
             )
             return False
 
-        if (
-            mapping_type == SlotMappingType.FROM_INTENT
-            and mapping.get(INTENT) not in domain.intents
-        ):
-            rasa.shared.utils.io.raise_warning(
-                f"Slot '{slot_name}' uses a 'from_intent' mapping for "
-                f"a non-existent intent '{mapping.get('intent')}'. "
-                f"Skipping slot extraction because of invalid mapping."
-            )
-            return False
+        if mapping_type is SlotMappingType.FROM_INTENT:
+            if isinstance(mapping.get(INTENT), list):
+                Mapping = mapping.get(INTENT)
+            else:
+                Mapping = [mapping.get(INTENT)]
+            
+            for intent in Mapping:
```

Comment:
This can be rewritten as suggested earlier by Lukas:
```suggestion
            intent_names = SlotMapping.to_list(mapping.get(INTENT))
            for intent in intent_names:
```